.github/workflows/set-milestone-from-base-branch.yml

```diff
@@ -38,23 +38,24 @@ jobs:
           URL_REGEX='(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)'
```

Comment:
Will this always work?

You can do
```
Resolves url-to-issue
```
or
```
Resolves #issueNumber
```
Wouldn't this fail in the latter case?

And even if we do always get a url, shouldn't we narrow it down to url patterns matching github issues?

Reply:
Sometimes people use urls to services other than GitHub after `Resolves`, like here https://github.com/brave/devops/pull/7762.